src/diffusers/models/unet_2d_blocks.py

```diff
@@ -383,6 +415,17 @@ def __init__(
                         norm_num_groups=resnet_groups,
                     )
                 )
+            elif attention_block == "AttentionBlock":
```

Comment:
this is too different for me, let's please not merge this all in the same Cross attention class. Here would create a new `UNetMidBlock2DCrossAttn` class name and add a new config param

Reply:
We can just add a new config variable with a class name:
```mid_block_type = "UNetMidBlockUnCLIPAttn"``` that will default to ```mid_block_type = "UNetMidBlock2DCrossAttn"```
=> this should fit nicely in the config here: https://huggingface.co/runwayml/stable-diffusion-v1-5/blob/main/unet/config.json#L14

Reply:
Yep, thank makes sense!

Reply:
done!